src/Pages/RegisterForm/ModalWindow/ModalWindow.tsx

```diff
@@ -1,11 +1,50 @@
-
-
-const ModalWindow = () => {
-  return (
-    <div>
-      
-    </div>
-  )
+import React from 'react';
+import {useNavigate} from 'react-router-dom';
+import { Button, Modal } from 'antd';
+import smile from '@/images/Homepage/smile.png';
+import './style.scss';
+interface ModalWindowProps{
+    isModalVisible:boolean,
+    handleModalClose:()=> void;
 }
+    
+
+const ModalWindow: React.FC<ModalWindowProps> = ({ isModalVisible,  handleModalClose }) => {
+    const navigate = useNavigate();
+  
+    const handleOk = () => {
+        handleModalClose(); 
+      navigate('/');
```

Comment:
use your paths constant instead of hardcoding the url. 
Try to ask question in this case- if at some point I will want to change my app's HOME path  from "/" to "/home"; in how many places i will have to change it ?  It is better f the answer is - in ONE single place.  

It is also referred to as DRY principle in coding - Don't Repeat Yourself.


```suggestion
      navigate(paths.HOME);
```

Reply:
Changed